ws-nextjs-app/pages/[service]/new_live/[id]/Posts/index.tsx

```diff
@@ -0,0 +1,95 @@
+import React from 'react';
+
+import pathOr from 'ramda/src/pathOr';
+import { OptimoBlock } from '#models/types/optimo';
+import headings from '#app/legacy/containers/Headings';
+import Blocks from '#app/legacy/containers/Blocks';
+import paragraph from '#app/legacy/containers/Paragraph';
+import Text from '#app/components/Text';
+import { Post, StreamResponse } from './post.d';
+
+// temporary solution to render LI/ OL blocks.
+const unorderedList = ({ blocks }: { blocks: OptimoBlock[] }) => {
+  const listItems: (string | null)[] = blocks
+    .map(item =>
+      pathOr<string | null>(
+        null,
+        ['model', 'blocks', 0, 'model', 'text'],
+        item,
+      ),
+    )
+    .filter(text => typeof text === 'string');
+
+  return (
```

Comment:
I'm thinking we should add a check to make sure we don't render an empty set of <list> tags.

```suggestion
  if(listItems.length > 0){
  return (
    <Text>
      <ul>
        {listItems.map(item => (
          <li>{item}</li>
        ))}
      </ul>
    </Text>
  );
  }
  
  return <> </>
```

Reply:
Good idea! I have tweaked this slightly so that it checks if the length is 0, and if it is then it returns null. Else it resturns the list.